pkgs/applications/terminal-emulators/kitty/default.nix

```diff
@@ -156,6 +157,7 @@ buildPythonApplication rec {
     substituteInPlace kitty_tests/file_transmission.py \
       --replace test_file_get dont_test_file_get \
       --replace test_path_mapping_receive dont_test_path_mapping_receive
+      --replace test_transfer_send
```

Comment:
```suggestion
      --replace test_path_mapping_receive dont_test_path_mapping_receive \
      --replace test_transfer_send dont_test_transfer_send
```

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/256072/commits/d13268ac8ae7e53cb7448eaf8f1538353c4481f4